packages/frontend/core/src/components/cloud/share-header-right-item/present.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from '@affine/component/ui/button';
+import { useAFFiNEI18N } from '@affine/i18n/hooks';
+import type { SurfaceService } from '@blocksuite/blocks';
+import { PresentationIcon } from '@blocksuite/icons';
+import { useCallback, useEffect, useState } from 'react';
+
+import * as styles from './styles.css';
+
+export const PresentButton = () => {
+  const t = useAFFiNEI18N();
+  const [isPresent, setIsPresent] = useState(false);
+
+  const handlePresent = useCallback(() => {
+    const editorRoot = document.querySelector('block-suite-root');
```

Comment:
do we have api from blocksute model?

Reply:
In the future, an editor's Atom will be encapsulated, so let's deal with it now.